src/content/apis/geocoding/geolocation .md

```diff
@@ -0,0 +1,10 @@
+---
+name: Geolocation
+description: IP2Location IP Geolocation web service is a REST API service to do a reverse IP location lookup for ISO3166 country code, region or state, city and so on.
+categories:
+  - geocoding
+features:
+  https: true
+url: https://www.geolocation.com/ip-geolocation-api
+logo: geolocation.jpg
```

Comment:
Why is this file named `geolocation .md` i.e. has a space before extension?? **Please fix it**

Also the image extension is incorrect so it is not loading.